src/Turing.jl

```diff
@@ -16,7 +16,7 @@ using Libtask
 using Tracker: Tracker
 
 import AdvancedVI
-import DynamicPPL: getspace, NoDist, NamedDist
+import DynamicPPL: getspace, NoDist, NamedDist, elementwise_loglikelihoods, generated_quantities
```

Comment:
Maybe use `using` to avoid accidentally overloading it and move the `using` and export statements to the re-export section (IIRC there was a comment about reexports somewhere in the file)?

Reply:
Yeah! Uhh, am I blind though? I can't find any comment on reexports in the file...

Reply:
Hmm seems I misremembered, it was in DynamicPPL: https://github.com/TuringLang/DynamicPPL.jl/blob/161f8200c5d608c481b58e056c2687a380f1c0d5/src/DynamicPPL.jl#L98-L100
Here we only use `@reexport` it seems. The question is - should we just `@reexport` DynamicPPL as well?

Reply:
See other comments